lib/optiontable.pl

```diff
@@ -96,25 +98,25 @@ sub add {
     }
 
     if(/^ *CURLOPT\(([^,]*), ([^,]*), (\d+)\)/) {
-        my($opt, $type, $num)=($1,$2,$3);
+        my ($opt, $type, $num) = ($1, $2, $3);
         add($opt, $type, $num);
     }
 
     # alias for an older option
     # old = new
     if(/^#define (CURLOPT_[^ ]*) *(CURLOPT_\S*)/) {
-        my ($o, $n)=($1, $2);
+        my ($o, $n) = ($1, $2);
+
         # skip obsolete ones
         if(($n !~ /OBSOLETE/) && ($o !~ /OBSOLETE/)) {
             $o =~ s/^CURLOPT_//;
             $n =~ s/^CURLOPT_//;
             $alias{$o} = $n;
-            push @names, $o,
+            push @names, $o,;
```

Comment:
```suggestion
            push @names, $o;
```

Reply:
do not seem to be able to fix this up to any satisfaction with perltidy (and trying hard to not add 2nd pass with sed)  - is status quo (leave unchanged) acceptable ?

Reply:
This one looks a fix to do manually.